### ISSUE TITLE
Project members repeated

### ISSUE THREAD

**Issue by marcalwestf** [adempiere/adempiere#2205]
It is possible to select the same user several times as a project member.
![20181210_project_13](https://user-images.githubusercontent.com/1789408/49822272-46112e00-fd42-11e8-9cba-391a3376e867.png)

A restriction should exist to allow only one entry per user in table _C_ProjectMember_.

**Comment by yamelsenih** [comment]
I agree with you @marcalwestf

**Comment by e-Evolution** [comment]
@marcalwestf yes is posible because can have a different member type. with same user.

**Comment by yamelsenih** [comment]
Hi @all i think that @e-Evolution comment it correct, 
I propose create a unique index like follow:

With a combination (C_Project_ID, AD_User_ID, C_MemberType_ID) for table C_ProjectMember

**Comment by yamelsenih** [comment]
Also is nice to have permission for **C_MemberType** definition

**Comment by marcalwestf** [comment]
@yamelsenih agree with spending a unique index as you suggest.

Without this unique index, it is possible to add twice the combination (user, member type) for a given project, which is an error.
